### ISSUE TITLE
tft38

### ISSUE THREAD

**Issue by vzagranichnyy** [miblooming/JZ-TS24-2#29]
Hi, I have jz-tsx-v1.4 tft 38. it has been idle for a long time, the stm32 processor burned out in it and I ordered a new processor, the wizard re-soldered it, but after turning on the white screen is on and it does not flash, I think you need a bootloader for it, could you help me with this?

**Comment by miblooming** [comment]
I'm sorry that bootloader is not open.

**Comment by vzagranichnyy** [comment]
> Извините, что загрузчик не открыт.

can I flash it?

**Comment by miblooming** [comment]
No. I have some old  jz-tsx board. jz-tsx is suspend production. I can give it to you as a gift, if you pay the shipping fee.

**Comment by vzagranichnyy** [comment]
> У меня есть старая доска JZ-TSX. JZ-TSX приостановить производство. Я могу подарить его вам, если вы оплатите стоимость доставки.

This is very generous.  I have a friend's address in Beijing.  Tell me how much to pay.  I can use web money or a visa card.

**Comment by miblooming** [comment]
To Beijing, shipping to you friend. And your friend  pay the shipping fee. Is it ok?

**Comment by vzagranichnyy** [comment]
> В Пекин, доставка вашему другу. И ваш друг оплатит стоимость доставки. Это нормально?

I will pay for the delivery myself.give your details.

**Comment by miblooming** [comment]
Connect next year. We are all on holiday for Chinese New Year before 2020-1-30.



Best Regards
LoomingXu


&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"vzagranichnyy"<notifications@github.com&gt;;
发送时间:&nbsp;2020年1月19日(星期天) 晚上7:25
收件人:&nbsp;"miblooming/JZ-TS24-2"<JZ-TS24-2@noreply.github.com&gt;;
抄送:&nbsp;"looming"<loomingxu@qq.com&gt;;"Comment"<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [miblooming/JZ-TS24-2] tft38 (#29)



 
В Пекин, доставка вашему другу. И ваш друг оплатит стоимость доставки. Это нормально?
  
I will pay for the delivery myself.give your details.
